community-operators/federatorai/federatorai.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,484 @@
+    mediatype: image/png
+  keywords: ['AI', 'Resource Orchestration', 'NoOps']
+  maintainers:
+  - email: info@prophetstor.com
+    name: ProphetStor Data Services, Inc.
+  provider:
+    name: ProphetStor Data Services, Inc.
+  links:
+  - name: FederatorAI
+    url: https://github.com/containers-ai/alameda
+  - name: FederatorAI Operator
+    url: https://github.com/containers-ai/federatorai-operator
+  labels:
+    alm-owner-federatorai: federatorai-operator
+    alm-status-descriptors: federatorai-operator.v0.0.1
+  selector:
+    matchLabels:
+      alm-owner-federatorai: federatorai-operator
+  customresourcedefinitions:
+    owned:
+    - name: alamedaservices.federatorai.containers.ai
+      version: v1alpha1
+      kind: AlamedaService
+      displayName: AlamedaService
+      description: An instance of Alameda.
+      resources:
+      - kind: Deployment
+        version: v1
+      - kind: ReplicaSet
+        version: v1
+      - kind: Pod
+        version: v1
+      specDescriptors:
+      - description: Enable Execution
+        displayName: Enable Execution
+        path: enableexecution
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+      - description: Enable dashboard
+        displayName: Enable GUI
+        path: enablegui
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: false
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
```

Comment:
Would this operator work with `permissions` in place of `clusterPermissions`? This operator only supports a SingleNamespace installMode, and so does not need to watch for CRs clusterwide.

Reply:
This operator intends to watch "AllNamepace" for AlamedaService CR so that it can deploy FederatorAI application in the namespace where CR is inside. So we change the installMode to also set AllNamespace to true. Would that be reasonable to use clusterPermission?

Reply:
This operator intends to watch "AllNamepace" for AlamedaService CR so that it can deploy FederatorAI application in the namespace where CR is inside. So we change the installMode to also set AllNamespace to true. Would that be reasonable to use clusterPermission?